contrib/haproxy/files/haproxy

```diff
@@ -0,0 +1,8 @@
+# haproxy service
+
+type           = bgprocess
+command        = /usr/bin/haproxy -f /etc/haproxy/haproxy.cfg -p /run/haproxy.pid
+pid-file       = /run/haproxy.pid
+logfile        = /var/log/haproxy.log
+run-as         = haproxy
```

Comment:
this can't work because dinit changes to this user before running the daemon. it then can't bind any (<1024) ports

normally instead you set

```
user        haproxy
group       haproxy
```

in the haproxy config

you can also name it `_haproxy` but not sure how much that matter